src/schemas/operations_log.yaml

```diff
@@ -70,9 +70,11 @@ write:
       description: If specified, indicates the change occurred over a span of time
       type: string
       format: iso8601-timestamp
+      nullable: true
```

Comment:
Is the idea here that if it's not set it will default to `CURRENT_TIMESTAMP`?

Reply:
Per the db column description:

> If specified, indicates the change occurred over a span of time

So it would just remain null.